### PR TITLE
double slabs didnt count the original slab 2 times

### PR DESCRIPTION
**Pull request by coehlrich**
this makes double slabs count the original half slab slab 2 times (also this is the first time i finished making something in java and not javascript so sorry if there are any errors)

### CODE REVIEW

**Comment by Lunatrius** [comment]
Did you try to compile the mod? I'm pretty sure that a closing curly bracket is missing as well as a cast to the actual slab type for the if check. I'll look into this later today.

**Comment by coehlrich** [comment]
just added the 2nd closing bracket now. i will try to compile it now

**Comment by coehlrich** [comment]
it seems to work now.

File: src/main/java/com/github/lunatrius/schematica/client/util/BlockList.java [hunk 1]
```diff
@@ -47,7 +47,16 @@
             final boolean isPlaced = BlockStateHelper.areBlockStatesEqual(blockState, mcBlockState);
 
             ItemStack stack = null;
-
+            
+            if (block.getIdFromBlock(block) == 43 || block.getIdFromBlock(block) == 125 || block.getIdFromBlock(block) == 181 || block.getIdFromBlock(block) == 204) {
```
Comment (Lunatrius) [inline src/main/java/com/github/lunatrius/schematica/client/util/BlockList.java]: This is definitely a no. I am against any special cases where possible, but using the block IDs directly is a definite no go.

I didn't have time to check this today, I'll check the code tomorrow.

**Comment by Lunatrius** [comment]
I don't want to sound rude, but did you actually check the double slab classes and it's methods?

As I said, I'll look into this tomorrow.

**Comment by coehlrich** [comment]
i did check the slab classes but for some reason block.isdouble didnt work

**Comment by coehlrich** [comment]
i may have found a alternative but i need the blockstate world and position (not sure why it needs the world and position when i am giving it the block state)

**Pull request closed without merging**